tests/e2e/utils/kubeflow_uninstallation.py

```diff
@@ -111,21 +111,16 @@ def delete_component(
                         delete_kustomize(kustomize_path)
 
                 elif component_name == "ingress":
-                    uninstall_helm(component_name, namespace)
-                    #Helm doesn't seem to delete ingress during uninstall
-                    exec_shell(f"kubectl delete ingress -n istio-system istio-ingress")
+                    if check_helm_chart_exists(component_name, namespace):
+                        uninstall_helm(component_name, namespace)
+                        #Helm doesn't seem to delete ingress during uninstall
+                        exec_shell(f"kubectl delete ingress -n istio-system istio-ingress")
                 else:
                     if check_helm_chart_exists(component_name, namespace):
                         uninstall_helm(component_name, namespace)
```

Comment:
should the ingress if condition be after this?
if chart exists -> uninstall helm -> if component = ingress -> delete using kubectl